### ISSUE TITLE
Apply tunings using lctl command

### ISSUE THREAD

**Issue by degremont** [cea-hpc/shine#202]
Use `lctl set_param` to apply tunings becomes more and more important. 
Doing this manually requires that Shine runs hundreds of commands in parallel which can create some issues.
And latest Lustre version (2.9+) is moving away from `/proc` and is using `/sys`. `lctl` makes this move transparent, taking care of writing in the correct place.

First things could be to support a new syntax in `tuning.conf`, along the current one.
``osc.*.max_dirty_mb=1024  NODELIST``

If this kind of lines is detected, Shine will run `lctl` instead of doing it with `echo`. This offers a smooth way to migrate.

It could be nice if the new syntax is compatible with either `ModelFile` or YAML syntax, to avoid maintaining another parser.

**Comment by tl-cea** [comment]
By doing this change, it would be nice to actually support any custom `lctl` command.

In particular, when many lnet routes have to be defined, they can't be passed as module arguments (limited to 1024 chars), so they have to be specified after loading lustre modules using `lctl add_route`.

Do you think this can be part of the same change, or should I open a specific ticket for that change request?

**Comment by degremont** [comment]
LNET config will be handled by a `lnetctl import` call. All LNET config will be moved into a YAML config file. See http://doc.lustre.org/lustre_manual.xhtml#idm140684903131360

A separate ticket will track support for this call.
